engine/src/main/java/org/terasology/engine/core/GameThread.java

```diff
@@ -18,14 +23,33 @@
  * </ul>
  *
  */
+@API
```

Comment:
Why did the `GameThread` become public API?

Reply:
do we need a separate API?

Reply:
see https://github.com/MovingBlocks/Terasology/pull/4799#issuecomment-904231869